src/plugins/home/public/application/components/homepage/section.tsx

```diff
@@ -65,28 +36,7 @@ export const Section: FC<Props> = ({ render, title, description, links }) => {
       color="transparent"
       data-test-subj="homepageSection"
     >
-      <EuiFlexGroup direction="row" alignItems="center" gutterSize="s" responsive={false}>
```

Comment:
The other reason i re-construct here is also because different sections now have a slightly different header: `recentWork` section has a filter button next to the toggle, `learn basics` section has a link next to the toggle. 

If we do not want to change the structure here at all, i guess we can add reactNode as another opt parameter for section, so they can choose to render that next to the toggle if they pass in a reactNode.

Reply:
Yep let's do that!